packages/firestore/src/definitions.ts

```diff
@@ -377,14 +377,6 @@ export interface SnapshotListenerOptions {
    * @default false
    */
   readonly includeMetadataChanges?: boolean;
-  /**
-   * Set the source the query listens to.
-   * The source `default` listens to both cache and server.
-   *
-   * @since 6.2.0
-   * @default "default"
-   */
-  readonly source?: 'default' | 'cache';
```

Comment:
Please revert this change. It's totally fine if this property is only supported on web. Just add a comment `Only available for Web.`.